src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,107 @@
+            var div3 = $(customtr).down().next().down();
+            div.style.display = "block";
+            div3.style.display = "block";
+
+            // make it visible
+            new Ajax.Request(this.getAttribute("helpURL"), {
+                method : 'get',
+                onSuccess : function(x) {
+                    var from = x.getResponseHeader("X-Plugin-From");
+                    div.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                    layoutUpdateCallback.call();
+                },
+                onFailure : function(x) {
+                    div.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                    layoutUpdateCallback.call();
+                }
+            });
+            
+            // Creation of unique id
+            var customHelpClassName = this.getAttribute("helpURL").replace(rootURL+"/", "").replace("/", ".");
+            while(customHelpClassName.indexOf("/") + 1){
+                customHelpClassName = customHelpClassName.replace("/", ".")
+            }
+
+            var customurl = rootURL + "/helpmanager/get?class="+customHelpClassName;
+                $(div2).setAttribute("customHelpUrl", rootURL+"/helpmanager/updateHelpInfo");
+                $(div2).setAttribute("className", customHelpClassName);
+
+            new Ajax.Request(customurl, {
+                method : 'get',
+                onSuccess : function(x) {
+                    var from = x.getResponseHeader("X-Plugin-From");
+                    div2.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                    layoutUpdateCallback.call();
+                },
+                onFailure : function(x) {
+                    div2.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                    layoutUpdateCallback.call();
+                }
+            });
+
+        } else {
+            var div = $(findFollowingTR(tr, "help-area")).down().next().down();
+            div.style.display = "none";
+            $(tr).remove();
+            layoutUpdateCallback.call();
+        }
+
+        return false;
+    };
+
+    e.tabIndex = 9999; 
+    e = null;
+        
+});
+
+function save_custom_help(myel) {
+    $(myel).style.display = "none";
+    $(myel).next().style.display = "block";
+    var div = $(myel).up().next();
+    div.innerHTML = div.down().value;
+
+    var xmlhttp;
+    if (window.XMLHttpRequest) {
+        xmlhttp = new XMLHttpRequest();
+    }
+    else {
+        xmlhttp = new ActiveXObject("Microsoft.XMLHTTP");
+    }
+
+    xmlhttp.open("POST",$(div).getAttribute("customHelpUrl"),true);
+    xmlhttp.setRequestHeader("Content-type","application/x-www-form-urlencoded");
+    xmlhttp.send("class="+$(div).getAttribute("className")+"&textArea="+div.innerHTML);
+
+    return false;
+};
+
+function edit_custom_help(myel) {
```

Comment:
This isn't idiomatic style really (minor nit). editCustomHelp() would look less out of place. (very minor).